test/user-profile/e2e/user-profile.e2e.spec.js

```diff
@@ -7,9 +7,11 @@ module.exports = E2eHelpers.createE2eTest(
     let token = LoginHelpers.getUserToken();
 
     LoginHelpers.logIn(token, client, '/profile', 1)
-      .waitForElementVisible('.loading-indicator-container', Timeouts.normal)
-      .waitForElementNotPresent('.loading-indicator-container', Timeouts.slow)
-//      .waitForElementVisible('.usa-button-big', Timeouts.normal)
+      // The loading indicator tests seem to be problematic because the thing goes away fairly quickly
```

Comment:
Is there a way to pass a parameter in that indicates this is running on Sauce, and only skip stuff if that's the case?